org.aposin.licensescout.core/src/main/java/org/aposin/licensescout/main/Main.java

```diff
@@ -55,18 +56,23 @@ private Main() {
      * The main method.
      * 
      * @param args the arguments
-     * @throws Exception 
+     * @throws Exception
      */
     public static void main(String[] args) throws Exception {
         File source;
         File target;
         LicenseStoreData licenseStoreData = null;
-        AbstractFinder finder = new JavaJarFinder(licenseStoreData, new JavaUtilLog(Logger.getGlobal()));
+        final JavaUtilLog log = new JavaUtilLog(Logger.getGlobal());
+        final RunParameters runParameters = new RunParameters();
+        // TODO: set parameters in RunParameters
```

Comment:
Please double check this. If it is a proper TODO open a ticket for it and add the number here. 
Otherwise just delete it.